### ISSUE TITLE
Add support for mapper 515

### ISSUE THREAD

**Issue by SuperMarioFan2007** [0ldsk00l/nestopia#396]
Is it possible to add NES 2.0 mapper 515 on Nestopia. Mapper 515 Family Noraebang

**Comment by carmiker** [comment]
Yes, it is entirely possible to add this mapper. I will add it to my list. I have recently added microphone input for Karaoke Studio and the FDS microphone, using a real physical microphone, to Nestopia JG, so this mapper could be one more thing that supports an emulated microphone through a physical microphone. The code for the YM2413 is more or less done in the Konami VRC7 code already, and if all else fails, I have a nice clean YM2413 implementation in one of my other emulators, Cega.
